Tella/Data/Database/TellaDataBase.swift

```diff
@@ -84,6 +83,22 @@ class TellaDataBase : DataBase {
         
     }
     
+    func checkFilesInReports(ids: [String]) -> Bool {
+        var isFileInReport = false
+        do {
+            try ids.forEach({ vaultId in
+                let condition = [KeyValue(key: D.cVaultFileInstanceId, value: vaultId)]
+                let responseDict = try statementBuilder.selectQuery(tableName: D.tReportInstanceVaultFile,
```

Comment:
I think this condition will check only the files in the reports, we need to add an other condition for Uwazi files